config/initializers/okcomputer.rb

```diff
@@ -6,9 +6,7 @@
   OkComputer::HttpCheck.new(Blacklight.default_index.connection.uri.to_s.sub(%r{/$}, '') + '/admin/ping')
 
 OkComputer::Registry.register 'downloads-cache',
-  OkComputer::DirectoryCheck.new(
-    Settings.DOWNLOAD_PATH || (Rails.root + 'tmp/cache/downloads').to_s, true
-  )
+  OkComputer::DirectoryCheck.new(Settings.DOWNLOAD_PATH, true)
 
```

Comment:
the `DOWNLOAD_PATH` should always exist now -- either the default in `settings.yml` below or the actual value from shared_configs